services-behaviors.md

```diff
@@ -90,6 +90,22 @@ Properties can be declared on an extendable object by calling `addDynamicPropert
     
 > **Note**: Attempting to set undeclared properties through normal means (`$this->foo = 'bar';`) on an object that implements the **October\Rain\Extension\ExtendableTrait** will not work. It won't throw an exception, but it will not autodeclare the property either. `addDynamicProperty` must be called in order to set previously undeclared properties on extendable objects.
 
+#### Retrieving dynamic properties
+
+Properties created dynamically can be retrieved with the getDynamicProperties function inherited from
+the ExtendableTrait.  
+
+So retrieving all dynamic property would look like this.
```

Comment:
"all dynamic property" to "all dynamic properties" please, period `.` to colon `:`